styx-scheduler-service/src/main/java/com/spotify/styx/Scheduler.java

```diff
@@ -378,9 +380,10 @@ private boolean hasTimedOut(RunState runState) {
     return !deadline.isAfter(now);
   }
 
-  private void sendTimeout(WorkflowInstance workflowInstance) {
-    LOG.info("Found stale state, issuing timeout for {}", workflowInstance);
-    stateManager.receiveIgnoreClosed(Event.timeout(workflowInstance));
+  private void sendTimeout(InstanceState instanceState) {
+    LOG.info("Found stale {} state, issuing timeout for {}",
```

Comment:
Can we print something more here? Maybe not the whole `runState` but at least the registered `runState().timestamp()` could be useful for debugging